quantecon/lss.py

```diff
@@ -9,10 +9,10 @@
 import numpy as np
 from numpy.random import multivariate_normal
 from scipy.linalg import solve
+from numba import jit
 
-#-Check if Numba is Available-#
-from .util import numba_installed, jit
-
+#TODO: Should this be specified with (nopython=True?)
```

Comment:
`@jit(nopython=True)` will work if the line `A = np.asarray(A)` is removed (assuming that the input `A` is an numpy ndarray). (But I suspect we wouldn't get much speedup.)